padertorch/contrib/examples/speech_enhancement/simple_mask_estimator/model.py

```diff
@@ -0,0 +1,89 @@
+import torch
+
+import padertorch as pt
+from padertorch.summary import mask_to_image, stft_to_image
+
+class SimpleMaskEstimator(pt.Model):
+    def __init__(self, num_features, num_units=1024, dropout=0.5,
+                 activation='elu'):
+        """
+
+        :param num_features: number of input features
+        :param num_units: number of units in linear layern
+        :param dropout: dropout forget ratio
+        :param activation:
```

Comment:
Use Google-Style docstring? A great part of the code in padertorch uses google style, so I think that new code that we add should also use this style

Reply:
done